modules/ROOT/nav.adoc

```diff
@@ -1,36 +1,37 @@
 .xref:index.adoc[Anypoint Monitoring]
```

Comment:
This organization is temporary but necessary for grouping like with like in time for the Dec release. It will better align with nav style guidelines in a future iteration after the release.